core/config/default-config.js

```diff
@@ -312,10 +306,6 @@ const defaultConfig = {
     'metrics': {
       title: str_(UIStrings.metricGroupTitle),
     },
-    'budgets': {
```

Comment:
What does an old report with budgets look like without this category? Are budget audits just hidden?